src/vue-components/reskin-font-color.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="alpheios-skin_panel alpheios-panel__options-item">
+  <div id="reskin-panel" class="alpheios-skin_panel alpheios-panel__options-item">
```

Comment:
@irina060981 Could you just make one more change here and add the alpheios namespace to the id? e.g. alpheis-reskin-panel? Since we can't control the pages in which Alpheios is embedded we need to try to make sure our ids don't clash. While it's not very likely that we will have a clash here, using the alpheios namespace is safer. Thanks!

Reply:
Oh - I didn't think about this aspect! I will add domain.